dist/flexboxgrid.css

```diff
@@ -1043,4 +1051,38 @@
     -ms-flex-order: 1;
     order: 1;
   }
+
+  .hide-lg {
+    display: none !important;
+  }
+}
+
+@media only screen and (max-width: 48em) {
+  .hide-xs {
+    display: none !important;
+  }
+}
+
+@media only screen and (min-width: 48em) and (max-width: 64em) {
+  .hide-sm {
+    display: none !important;
+  }
+}
+
+@media only screen and (min-width: 64em) and (max-width: 75em) {
+  .hide-md {
+    display: none !important;
+  }
+}
+
+@media only screen and (max-width: 64em) {
+  .hide-sm-down {
+    display: none !important;
+  }
+}
+
+@media only screen and (max-width: 75em) {
```

Comment:
all of these `max-width`s should probably be more like 74.9375em otherwise they apply too far up?